test/readme.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-env node, mocha */
+
+var expect = require('chai').expect;
+var Exonum = require('..');
+
+describe('Examples from README.md', function() {
+
+    it('should sign custom type', function() {
+        var User = Exonum.newType({
+            size: 9,
+            fields: {
+                name: {type: Exonum.String, size: 8, from: 0, to: 8},
+                age: {type: Exonum.Int8, size: 1, from: 8, to: 9}
+            }
+        });
+        var data = {
+            name: 'Tom',
+            age: 34
+        };
+        var secretKey = '978e3321bd6331d56e5f4c2bdb95bf471e95a77a6839e68d4241e7b0932ebe2bfa7f9ee43aff70c879f80fa7fd15955c18b98c72310b09e7818310325050cf7a';
+        var signature = Exonum.sign(secretKey, data, User);
+
+        expect(signature).to.equal('c1db9a5f01ebdff27e02652a9aae5c9a4ac88787587dabceb9f471ae0b8e051b9632dfd26922f6abf24ff2d3275028fe286703d25ee7fe6b1711e89af4a7d307');
+    });
+
+    it('should verify custom type signature', function() {
+        var User = Exonum.newType({
+            size: 9,
+            fields: {
+                name: {type: Exonum.String, size: 8, from: 0, to: 8},
+                age: {type: Exonum.Int8, size: 1, from: 8, to: 9}
+            }
+        });
+        var data = {
+            name: 'Tom',
+            age: 34
+        };
+        var publicKey = 'fa7f9ee43aff70c879f80fa7fd15955c18b98c72310b09e7818310325050cf7a';
+        var signature = 'c1db9a5f01ebdff27e02652a9aae5c9a4ac88787587dabceb9f471ae0b8e051b9632dfd26922f6abf24ff2d3275028fe286703d25ee7fe6b1711e89af4a7d307';
+
+        expect(Exonum.verifySignature(signature, publicKey, data, User)).to.equal(true);
```

Comment:
Nitpick: you may use `.to.be.true` (or `to.be.true()` if you use [`dirty-chai`](https://www.npmjs.com/package/dirty-chai) to get rid of possible linter warnings).